apps/angular/src/shared/components/input-password/input-password.component.html

```diff
@@ -0,0 +1,10 @@
+<mat-form-field subscriptSizing="dynamic" appearance="outline" class="form-field">
+	<mat-icon matIconPrefix>lock</mat-icon>
+	<input matInput [type]="hidePassword() ? 'password' : 'text'" [placeholder]="placeholder" [formControl]="control">
+	<button mat-icon-button matSuffix type="button" (click)="clickHidePassword($event)">
```

Comment:
```suggestion
	<button mat-icon-button matSuffix type="button" (click)="onHidePasswordClick($event)">
```
https://wiki.saritasa.rocks/frontend/languages/ts-js/naming#event-handlers